appinventor/misc/translate/script.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#########################################################
+###     OdeMessages.java converter to .properties     ###
+###   To add compatibility with translation centers   ###
+###          Recommended: OneSky, Transifex           ###
+### ------------------------------------------------- ###
+###     Copyright 2017 - Diego Barreiro, Makeroid     ###
+#########################################################
+
+
+lines = open('../../appengine/src/com/google/appinventor/client/OdeMessages.java', "r", encoding="utf-8").read().split("\n")
+out = open('OdeMessages.properties', 'w', encoding="utf-8")
+
+for line in lines:
+    if line.startswith("  @DefaultMessage"):
+        DefaultMessage = line.split('"')
```

Comment:
The logic here assumes that one never uses a `"` within a string by escaping it. It also doesn't account for messages that span multiple lines using string literal concatenation. I would expect the following example to break:

```java
@DefaultMessage("The following \"thing\" is broken:\n" +
   "Exception: {0}")
```